docs/examples/sequence_processing/video/video_reader_simple_example.ipynb

```diff
@@ -47,7 +47,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is an mp4 container containing an h264 video and distributed under the Create Common license. Let's split it into 10s clips in order to check how `VideoReader` handles mutliple video files. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. The snippet below verifies, that you have defined `DALI_extra` path as an environment variable."
+    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is an mp4 container containing an h264 video and distributed under the Create Common license. Let's split it into 10s clips in order to check how `readers.Video` handles mutliple video files. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. The snippet below verifies, that you have defined `DALI_extra` path as an environment variable."
```

Comment:
```suggestion
    "We need some video containers to process. We can use [Sintel](https://en.wikipedia.org/wiki/Sintel) trailer, which is an mp4 container containing an h264 video and distributed under the Create Common license. Let's split it into 10s clips in order to check how `readers.Video` handles multiple video files. This can be done easily with the `ffmpeg` standalone tool. `DALI_EXTRA_PATH` environment variable should point to the place where data from [DALI extra repository](https://github.com/NVIDIA/DALI_extra) is downloaded. Please make sure that the proper release tag is checked out. The snippet below verifies, that you have defined `DALI_extra` path as an environment variable."
```

Reply:
done